app.rb

```diff
@@ -0,0 +1,32 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    "<h1 style='text-align:center'>Welcome to Chitter</h1>"
```

Comment:
It's good practice to pull this into its own era file, to adhere to a separation between the Model <-> Controller <-> View. 

Partially because it's easier for a dev to navigate directly to a view file in a Views folder to make changes to the UI, without needing to read through & alter the controller.